<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-O54SLV8" failures="1" tests="3" name="Functional Test on edge" time="123.435" errors="0" timestamp="2025-06-23T02:55:03 EEST">
  <testcase classname="Tests.TC1_EnterProductName" name="InValid_Search_TC1" time="18.311">
    <failure type="java.lang.AssertionError" message="&amp;apos;No results message&amp;apos; is not displayed for invalid product search expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: 'No results message' is not displayed for invalid product search expected [true] but found [false]
at Tests.TC1_EnterProductName.InValid_Search_TC1(TC1_EnterProductName.java:75)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- InValid_Search_TC1 -->
  <testcase classname="Tests.TC2_AddingItemsToCart" name="Add_Item_to_cart_TC" time="13.89"/>
  <testcase classname="Tests.TC3_CheckoutProcess" name="Valid_Checkout" time="22.445"/>
</testsuite> <!-- Functional Test on edge -->
